src/Providers/MiddlewareProvider.php

```diff
@@ -28,5 +32,8 @@ public function register()
 
             return new MultiTenantFamilyHydrator($entityManager->getRepository(Family::class));
         });
+
+        $this->container->add(PaginationParser::class);
```

Comment:
Is there a strong reason not to new these as we need them?

Reply:
Nope, actually removing this in the PR I'm working on now. :)